docs/products/kafka/kafka-mirrormaker/howto/log-analysis-offset-sync-tool.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Apache Kafka® MirrorMaker 2 log analysis using the offset sync tool
+sidebar_label: Offset sync log analysis
+---
+
+Analyze key log messages during data synchronization with Apache Kafka® MirrorMaker 2 using Aiven’s offset sync inspection tool.
```

Comment:
By "log messages" do you mean like slf4j/printf/etc text streaming out of the services as they're running? Or Kafka topics/event streams/etc?

If it's the former, this title is inaccurate as this tool has nothing to do with debug logging.
If it's the latter, I think "log analysis" is a confusing term, as it makes me think of text logging.

Reply:
Thanks, @gharris1727 ! 
I don’t have the complete setup to check this tool's output. 
Could you confirm if this tool's output is offset synchronization data or status information rather than traditional debug logs (e.g., `slf4j` or `print` logs)?
- Would it be more accurate to refer to the tool’s output as "offset sync data" or "synchronization status" instead of “log messages”?
- Similarly, would a title like “Offset Sync Status Analysis” or “Offset Synchronization Review” better reflect the tool’s purpose?

cc: @MichaelTansiniAiven

Reply:
Update the title and intro sections.